z.mlip

```diff
@@ -659,28 +659,29 @@ external (/<): t -> t -> t = "ml_z_fdiv"
 external (/|): t -> t -> t = divexact@ASM
```

Comment:
I think that also needs to be changed here.

```suggestion
external (/|): t -> t -> t = "ml_z_divexact"
```

Reply:
Well spotted, thank you!